packages/app-content-pages/src/screens/Publications/components/Publication/Publication.js

```diff
@@ -8,43 +8,56 @@ const StyledBox = styled(Box)`
   overflow: hidden;
 `
 
-const Placeholder = (
-  <Box
-    align='center'
-    justify='center'
-    background='brand'
-    height='100%'
-    width='100%'
-  >
-    <ZooniverseLogo size='50%' />
-  </Box>
-)
+const Placeholder = ({ id }) => {
+  return (
+    <Box
+      align='center'
+      justify='center'
+      background='brand'
+      height='100%'
+      width='100%'
+    >
+      <ZooniverseLogo size='50%' id={id} />
+    </Box>
+  )
+}
 
-function Publication (props) {
-  const { authors, avatarSrc, className, title, url, year } = props
+function Publication({
+  authors = '',
+  avatarSrc = '',
+  className = '',
+  title = '',
+  url = '',
+  year = ''
+}) {
   const displayString = [title, authors, year].filter(v => v).join(', ')
 
   return (
     <Box
       className={className}
+      data-testid='publication-test-element'
       direction='row'
       gap='small'
       margin={{ bottom: 'small' }}
     >
       <StyledBox height='50px' width='50px' flex={false}>
-        {!avatarSrc && Placeholder}
+        {!avatarSrc && <Placeholder id={displayString} />}
```

Comment:
I think this is an invalid identifier.

Reply:
>Technically, the value for an id attribute may contain any character, except [whitespace](https://developer.mozilla.org/en-US/docs/Glossary/Whitespace) characters. However, to avoid inadvertent errors, only [ASCII](https://developer.mozilla.org/en-US/docs/Glossary/ASCII) letters, digits, '_', and '-' should be used, and the value for an id attribute should start with a letter.

https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/id

Reply:
The accessibility checker in the storybook catches invalid IDs. That's useful.

<img width="300" alt="The AXE accessibility checker in the storybook, warning that `id` must be unique on the Publications page." src="https://github.com/zooniverse/front-end-monorepo/assets/59547/826019f2-96f8-4a05-a0c1-189078b3b1a0">

Reply:
Thanks! I'll refactor this prop